lib/active_merchant/billing/gateways/commerce_hub.rb

```diff
@@ -190,14 +193,9 @@ def build_purchase_and_auth_request(post, money, payment, options)
 
       def add_reference_transaction_details(post, authorization, options, action = nil)
         reference_details = {}
-        merchant_reference, transaction_id = authorization.include?('|') ? authorization.split('|') : [nil, authorization]
-
-        if action == :refund
-          reference_details[:referenceTransactionId] = transaction_id
-        else
-          reference_details[merchant_reference.present? ? :referenceMerchantTransactionId : :referenceTransactionId] = merchant_reference.presence || transaction_id
-        end
+        _merchant_reference, transaction_id = authorization.include?('|') ? authorization.split('|') : [nil, authorization]
```

Comment:
Small nit: `_` before the var isn't really necessary, tho I understand this is more of a Pythonic way to denote an unused variable. In Ruby, you can just stub the `_` by itself for unused variables.